info/guide/diktat-coding-convention.md

```diff
@@ -31,35 +31,35 @@ I [Preface](#c0)
     * [2.4.1 Add a blank line between the body of the comment and Kdoc tag-blocks](#r2.4.1)
     * [2.4.2 Do not comment on unused code blocks](#r2.4.2)
     * [2.4.3 Code delivered to the client should not contain TODO/FIXME comments](#r2.4.3)
-    
+
 [3. General formatting (typesetting)](#c3)
 * [3.1 File-related rules](#c3.1)
     * [3.1.1 Avoid files that are too long](#r3.1.1)
     * [3.1.2 Code blocks in the source file should be separated by one blank line](#r3.1.2)
-    * [3.1.3 Import statements order](#r3.1.3)  
-    * [3.1.4 Order of declaration parts of class-like code structures](#r3.1.4)      
+    * [3.1.3 Import statements order](#r3.1.3)
+    * [3.1.4 Order of declaration parts of class-like code structures](#r3.1.4)
     * [3.1.5 Order of declaration of top-level code structures](#r3.1.5)
-* [3.2 Braces](#c3.2)    
-    * [3.2.1 Using braces in conditional statements and loop blocks](#r3.2.1)           
-    * [3.2.2 Opening braces are placed at the end of the line in *non-empty* blocks and block structures](#r3.2.2)               
-* [3.3 Indentation](#c3.3)            
-* [3.4 Empty blocks](#c3.4) 
-* [3.5 Line length](#c3.5) 
-* [3.6 Line breaks (newlines)](#c3.6) 
-    * [3.6.1 Each line can have a maximum of one statement](#r3.6.1) 
-    * [3.6.2 Rules for line-breaking](#r3.6.2) 
+* [3.2 Braces](#c3.2)
```

Comment:
please add a readme for these gradle tasks

Reply:
Done - added informative README.md in Info directory